tests/torch/test_hybrid_converter.py

```diff
@@ -52,11 +57,16 @@ def run_hybrid_llm_test(
         compress_input_ciphertexts=True,
     )
 
+    logits_simulate = None
+
     with monkeypatch.context() as m:
         if not transformers_installed:
             m.setitem(sys.modules, "transformers", None)
             if has_pbs_reshape:
                 has_pbs = True
+        if not glwe_backend_installed:
```

Comment:
this simulates environments where `concrete-ml-extensions` is not available (e.g. MAC)